src/Pim/Bundle/CatalogBundle/Elasticsearch/ProductIndexer.php

```diff
@@ -64,6 +66,33 @@ public function indexAll(array $products, array $options = [])
         }
 
         $this->indexer->bulkIndexes($this->indexType, $normalizedProducts, 'id');
+
+        if (!isset($options['associated_products']) || false === $options['associated_products']) {
```

Comment:
Ah ok, you should definitely put this information in the phpdoc. This is important.